.changelog/1400.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/cloudflare_logpush_job: Fix `dataset` argument value of `dns_logs` not being supported
+```
```

Comment:
this isn't a bug but an enhancement now that it is supported publicly

~~~suggestion
```release-note:enhancement
resource/cloudflare_logpush_job: add support for managing`dns_logs`
```
~~~

Reply:
ahh true!